chain/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+name = "cardano-multiplatform-lib-chain"
+version = "0.1.0"
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+cardano-multiplatform-lib-core = { "path" = "../core" }
+cardano-multiplatform-lib-crypto = { "path" = "../crypto" }
+cbor_event = "2.2.0"
+linked-hash-map = "0.5.3"
```

Comment:
Some of the dependencies will need to be cleared up and such but we can do that after we finalize the workspace structure.